### DISCUSSION TITLE
Time tracking report, August 2022

### DISCUSSION THREAD

**Discussion by aurelienpierre** [aurelienpierreeng/ansel#36] [Time tracking and invoices]
As announced on the wiki, this is the first report of the working time spent on R&Darktable, with associated costs. These reports will be made monthly to track how the resource is spent and how much it actually costs to make "free" software and maintain it.

The invoice can be found there : https://aurelienpierreengineering.harvestapp.com/client/invoices/ab37782ef336090a771348f040d8ffbe9e2b29ed

**33.5 h have been spent, for a total cost of 837 €.** [Liberapay donations](https://liberapay.com/aurelienpierre/) have covered 910 € before Stripe/Paypal transfer fees, which amounts to approximately 855 € after all fees *(Note : transfer fees have a flat and a proportional part, plus currency conversion fees on top if applicable, so I simply round everything to a flat 6% average fee)*.

11 h have been spent on #29, which amounts to 100% of the programming/debugging time this month. The associated cost so far is 275 € and the refactoring is not finished yet. This is the cost of cleaning up the mistakes programmed by hobbyists for free, and I want to be very clear on the fact that this free work is starting to get expensive.

17.45 h and 436 € have been spent on marketing, which corresponds to analysing the results of the 2020 darktable user survey and writing the Python code to clean, process and plot the data as well as write the report post. This work is essential since no other data is available on this topic and designing blindly while fantasizing the "average user" needs to stop in FLOSS.

**Comment by sahlirc** [comment]
I think that this is a good entry for donations in addition to be informative on the Ansel's workload by the developers.

I am old-fashioned, but believe that when a first beta version is available it should be rewarded by a donation. Unfortunately liberapay is now closed. Please do something about that (not necessarily at an individual level, but at a project level as in public research), as I believe that with the hard work having been done recently, 0.1 is not that far.

**Reply by aurelienpierre** [reply]
Donations are the worst model of financing because they come with no responsibilities. The solution is a cooperative. See https://ansel.photos/en/contribute/democracy/ The paradigm of "reward for work done" is unsustainable, the only way forward is "resources to work", which means securing budget upfront.
